src/main/java/frc/robot/RobotContainer.java

```diff
@@ -159,9 +160,13 @@ private void configureBindings() {
     //     .withTargetDirection(Rotation2d.fromDegrees(270));
     // }));
 
-    // reset the field-centric heading
-    driverJoystick.povCenter().onTrue(drivetrain.runOnce(() -> drivetrain.setNewPose(new Pose2d(0,0,new Rotation2d(0)))));
+    // reset the field-centric heading on left trigger press
+    driverJoystick.y().onTrue(drivetrain.runOnce(() -> drivetrain.resetRotation(new Rotation2d(DriverStation.getAlliance().equals(Alliance.Blue) ? 0 : 180))));
 
+    // reset fused vision pose estimator on left bumper press
+    driverJoystick.povCenter().onTrue(drivetrain.runOnce(() -> drivetrain.resetPose(drivetrain.getDrivePoseEstimator().getEstimatedPosition())));
```

Comment:
I would have `resetPose` reset the pose of both the fused and the odometry-only pose, and have a separate function call resetFusedPose that only resets the fused (or just entirely encapsulate that functionality and have a `setFusedPoseToOdometryPose` function)